todoapp/users/serializers.py

```
@@ -1,10 +1,48 @@
-from django.contrib.auth import authenticate
-from django.contrib.auth import get_user_model
+from django.contrib.auth import authenticate, get_user_model
```

Ruff (0.12.2)

`django.contrib.auth.authenticate` imported but unused

Remove unused import

(F401)